Upstream-Status: Pending

Fix -Werror=format-security in syslog call.

diff --git a/stbup/stbup.c b/stbup/stbup.c
index 7fd6b78..0000000 100644
--- a/stbup/stbup.c
+++ b/stbup/stbup.c
@@ -100,9 +100,9 @@ void myDebug(const char* str, ...)
         va_list ap;
         va_start(ap, str);
         vsnprintf(buf, 1024, str, ap);
         va_end(ap);
 
         printf("%s\n",buf);
         if(debug>1)
-                syslog(LOG_USER | LOG_DEBUG, buf);
+                syslog(LOG_USER | LOG_DEBUG, "%s", buf);
     }
 }
