From 1c52f6f93102397966c601f96799013f2b62aebb Mon Sep 17 00:00:00 2001
From: Thilo Graf <dbt@novatux.de>
Date: Sun, 12 Mar 2023 19:19:27 +0100
Subject: [PATCH] input: Makefile.am: remove install-exec-local

Breaks build
---
 Makefile.am | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 2a3061b..8c2abc6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4,7 +4,6 @@ AM_CPPFLAGS =  \
 
 bin_PROGRAMS = input
 
-
 input_SOURCES = \
 	input.c \
 	inputd.c \
@@ -18,8 +17,5 @@ input_SOURCES = \
 	png_helper.cpp
 
 input_LDADD = \
-	@FREETYPE_LIBS@
-
-install-exec-local:
-	install -d $(DESTDIR)/usr/bin
-	install -m 0755 input $(DESTDIR)/usr/bin/input
+	@FREETYPE_LIBS@ \
+	@PNG_LIBS@
-- 
2.30.2

